packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -428,6 +431,9 @@ class _ModalBottomSheetRoute<T> extends PopupRoute<T> {
 /// that a modal [BottomSheet] needs to be displayed above all other content
 /// but the caller is inside another [Navigator].
 ///
+/// The `isDismissible` parameter specifies whether the bottom sheet will be
```

Comment:
Using square brackets create a hyperlink to the parameter in question in the API documentation.
```suggestion
/// The [isDismissible] parameter specifies whether the bottom sheet will be
```

Reply:
Done.